tests/migrations/test_operations.py

```diff
@@ -3588,6 +3624,84 @@ def test_remove_index(self):
         self.unapply_operations("test_rmin", project_state, operations=operations)
         self.assertIndexExists("test_rmin_pony", ["pink", "weight"])
 
+    def test_alter_model_bases(self):
+        """
+        Tests the AlterModelBases operation.
+        """
+
+        project_state = self.set_up_test_model("test_almotb", mti_model=True)
+        operation = migrations.AlterModelBases(
+            "shetlandpony",
+            bases=(
+                models.Model,
+                Mixin,
+            ),
+        )
+        self.assertEqual(
+            operation.describe(),
+            "Update shetlandpony bases to {}".format(
+                (
+                    models.Model,
+                    Mixin,
+                )
+            ),
+        )
+
+        # Test the state alteration
```

Comment:
To add more explanation, docstrings and comments should state the expected behavior and omit prefixes like "Tests" since all tests test things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).